src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -200,3 +199,40 @@ const IntegrationFlex = styled('div')`
 const IntegrationItemBox = styled('div')`
   flex: 1;
 `;
+
+const IntegrationStatus = styled(
+  (props: React.HTMLAttributes<HTMLElement> & {status: ObjectStatus}) => {
+    const {status, ...p} = props;
+    const color = status === 'active' ? theme.success : theme.gray2;
+    const titleText =
+      status === 'active'
+        ? 'This Integration can be disabled by clicking the Uninstall button'
```

Comment:
@NisanthanNanthakumar From what I learned yesterday, the process of picking up translations involves finding instances of `t('...')`, so making the text a variable will cause us to not pick up the translation. I also know that I personally was doing this wrong many times before but better to start doing it right now.